src/main.js

```diff
@@ -16,17 +16,32 @@ const checkValidationOfValues = envSpecString => {
     "week"
   ];
   const alphanumericThatDoesNotStartWithDigit = /^[A-Z_][0-9A-Z_]*$/;
+  const genericForCheckingRestrChoicesSyntax = /^\[(.*)\]$/;
   let envValArray = parseVarFromType(envSpecString.trim().split("\n")); //split lines based on \n character and parse them
```

Comment:
Can you rename this to something more semantic? Now that I have to read more code, I struggle every time understanding what's inside this variable.

What about `envSpecLines` or something like that?